README.md

```diff
@@ -18,6 +18,7 @@ The simplest solution for pagination in Angular.
 * [Multiple Instances](#multiple-instances)
 * [FAQ](#faq)
   + [Why does my filter not work with pagination?](#why-does-my-filter-not-work-with-pagination)
+  + [How to use the ngFor index with pagination pipe?](#how-to-use-the-ngfor-index-with-pagination-pipe)
```

Comment:
This could be phrased in a more natural-sounding way like this:

"How do I use the ngFor index with the pagination pipe?" (and same goes for the section heading.